core/src/main/java/io/grpc/ClientInterceptors.java

```diff
@@ -141,7 +141,9 @@ public void cancel() { }
     public void halfClose() { }
 
     @Override
-    public void sendPayload(Object payload) { }
+    public boolean sendPayload(Object payload) {
+      return true;
```

Comment:
Treat it the same as closed. If one returns true, then the other should return true. If false, false.